packages/node/test/behavior/state/validation/conformanceTest.ts

```diff
@@ -466,12 +467,12 @@ const AllTests = Tests({
 
                 "disallows disallowed": {
                     record: { test: 3 },
-                    error: disallowedEnum("X", "disallowed", 3),
+                    error: disallowedEnum("all", "disallowed", 3),
```

Comment:
This is testing whether conformance "X" works, so presumably it should still be a conformance error?  Like, we don't want "X" to throw a different error for different field types?